LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2020 Pradyun Gedam
+Copyright (c) 2021 Pradyun Gedam and Contributors
```

Comment:
(I am not a licensed attorney, and the following should not be taken as formal legal advice.)

The original line does need to be included, but the problem with multiple copyright lines of text, rather than the single one matching the form of the one specified as variable in the [SPDX definition of the MIT license](https://spdx.org/licenses/MIT.html), often breaks programmatic license detection and validation, including GitHub's. A common alternative would be placing both complete copyright notices on the same line with clear lexical separation between them, like this:

```suggestion
Copyright Rebecca Turner <me@re-becca.org>; Copyright (c) 2021 Pradyun Gedam and Contributors
```

Given the original copyright notice is preserved and set off with a semicolon, both follow the form of standalone copyright notices, and this one line is permitted to vary without changing the legal effect of the license, it would seem equivalent separation is achieved without resorting to a line break.